shell/src/main/java/alluxio/cli/fs/command/CopyFromLocalCommand.java

```diff
@@ -66,7 +65,10 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
 
   @Override
   public String getUsage() {
-    return "copyFromLocal [-t <number of threads for copying>] <src> <remoteDst>";
+    return "copyFromLocal "
+        + "[-t <number of threads for copying>] "
+        + "[-bs <read buffer size in bytes>] "
```

Comment:
nit: `--buffersize` than `-bs` to be more informative

Reply:
changed to long option.